mentoring-app/src/app/models/models.ts

```diff
@@ -0,0 +1,27 @@
+export interface Course {
+    id: number,
+    title: string,
+    creation_date: string,
+    duration: number,
+    description: string
+  }
+  
+ export interface User {
+    id: number,
+    first_name: string,
+    last_name: string
+  }
+
+  class TestCourse implements Course {
```

Comment:
Rewrite classes so that they could be instantiated with appropriate data.

Reply:
done